packages/client/src/object/aggregateOrThrow.test.ts

```diff
@@ -226,22 +230,22 @@ describe("aggregateOrThrow", () => {
               id: {
                 type: "double";
               };
+              locationCity: {
+                type: "string";
+              };
             };
           };
         };
         actions: {};
         queries: {};
-      }["objects"]["Todo"],
-      {
-        locationCity: "approximateDistinct";
-        text: "approximateDistinct";
-      }
+      }["objects"]["Todo"]
     > = {
       select: {
         locationCity: "approximateDistinct",
+        text: "approximateDistinct",
       },
-    } as any;
+    };
 
-    expectType<"approximateDistinct">(f.select.locationCity);
+    // expectType<"approximateDistinct">(f.select.locationCity);
```

Comment:
Flagging whether this needs to get re-enabled now (or after the next PR merges perhaps?)

Reply:
Yea I think this would be after next PR